doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1884,6 +1884,7 @@ The most important system services of a security server are as follows.
  `xroad-jetty `     | Application server running the user interface | `/var/log/xroad/jetty/`
  `xroad-proxy`      | Message exchanger                             | `/var/log/xroad/proxy.log`
  `xroad-signer`     | Manager process for key settings              | `/var/log/xroad/signer.log`
+ `xroad-proxy-ui-api`  | Management UI and REST API                 | `/var/log/xroad/proxy_ui_api.log` and <br/>`/var/log/xroad/proxy_ui_api_access.log` 
```

Comment:
The version number of the doc should be incremented too

Reply:
fixed